File: rb/test/exists_test.rb
# Thesmsworks SDK exists test

require "minitest/autorun"
require_relative "../Thesmsworks_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ThesmsworksSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: csharp/test/ExistsTest.cs
// Thesmsworks SDK exists test.

using Xunit;

using ThesmsworksSdk;

namespace ThesmsworksSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = ThesmsworksSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
